File: src/main/java/com/lineying/bean/EmailMessage.java
package com.lineying.bean;

/**
 * 邮件消息
 */
public class EmailMessage {
    String senderAccount;
    String receiverAccount;
    String senderName;
    String subject;
    String content;

    public EmailMessage() {

    }

    public EmailMessage(String senderAccount, String receiverAccount, String senderName, String subject, String content) {
        this.senderAccount = senderAccount;
        this.receiverAccount = receiverAccount;
        this.senderName = senderName;
        this.subject = subject;
        this.content = content;
    }

    public String getSenderAccount() {
        return senderAccount;
    }

    public void setSenderAccount(String senderAccount) {
        this.senderAccount = senderAccount;
    }

    public String getReceiverAccount() {
        return receiverAccount;
    }

    public void setReceiverAccount(String receiverAccount) {
        this.receiverAccount = receiverAccount;
    }

    public String getSenderName() {
        return senderName;
    }

    public void setSenderName(String senderName) {
        this.senderName = senderName;
    }

    public String getSubject() {
        return subject;
    }

    public void setSubject(String subject) {
        this.subject = subject;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }


}

File: src/main/java/com/lineying/bean/EmailMessage.kt
package com.lineying.bean

/**
 * 邮件消息
 */
class EmailMessage {
    var senderAccount: String? = null
    var receiverAccount: String? = null
    var senderName: String? = null
    var subject: String? = null
    var content: String? = null

    constructor()
    constructor(senderAccount: String?, receiverAccount: String?, senderName: String?, subject: String?, content: String?) {
        this.senderAccount = senderAccount
        this.receiverAccount = receiverAccount
        this.senderName = senderName
        this.subject = subject
        this.content = content
    }

}